pkg/go.mod

```diff
@@ -46,7 +46,7 @@ require (
 	github.com/xeipuuv/gojsonschema v1.2.0
 	github.com/zclconf/go-cty v1.13.2
 	// DO NOT UPDATE gocloud.dev until https://github.com/pulumi/pulumi/issues/11986 is resolved
-	gocloud.dev v0.27.0
+	gocloud.dev v0.28.0
```

Comment:
Shouldn't this still be 27?

Reply:
Yes, I'm just upgrading to 0.28 to make sure the test is failing in CI, but I'll revert that before taking the PR out of draft mode.